03-pipefilter.md

```diff
@@ -79,119 +86,86 @@ $ wc *.pdb
 > themselves. It is the shell, not the other programs, that deals with
```

Comment:
In line 84, `wc`is used, but it has not been used before in your modified lesson. This may confuse learners.